### ISSUE TITLE
Support bin/setup convention

### ISSUE THREAD

**Issue by pmackay** [openfoodfoundation/openfoodnetwork#869]
This seems like a good practice:

https://robots.thoughtbot.com/bin-setup

**Comment by mkllnk** [comment]
I wrote a script for that, a long while ago. The pull request was around for more than a year, but we decided not to do it. The reasons against were:

- duplicate content of readme and install.sh
- install.sh maybe not reliable on different distributions
- confusion about how to deploy the project on a server

But I still think that a setup script could be good. The main question is: Do we want people to go through the setup steps manually and have a deeper understanding of what is involved? Or do we want people to save time by executing a script?

In that script I had some checks like

- is postgres installed?
- has psql the right version?
- is the right ruby version set?
